src/hw6/Car.java

```diff
@@ -20,4 +20,4 @@ private void startCommand() {
     private void startFuelSystem() {
         System.out.println("Start fuel system");
     }
-}
+} 
```

Comment:
Please format all your classes in this homework. IDEA can help you with that just press CTRL+ALT+L on a chosen class